app/src/components/CopyToast.tsx

```
@@ -0,0 +1,27 @@
+  return (
+    <div class="fixed bottom-4 right-4 bg-[#FFFFFF] dark:bg-[#252525] text-gray-900 dark:text-[#FFFFFF] rounded-lg p-3 shadow-lg border border-gray-200 dark:border-gray-800 flex items-center space-x-2 animate-fade-in">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        class="h-5 w-5 text-green-500"
+        viewBox="0 0 24 24"
+        fill="none"
+        stroke="currentColor"
+        stroke-width="2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+      >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)